lambo

```diff
@@ -314,8 +323,8 @@ fi
 PROJECTURL="http://$PROJECTNAME.$TLD"
 perlCommands=(
     "s/(DB_DATABASE=)(.*)/\1$PROJECTNAME/g"
-    's/(DB_USERNAME=)(.*)/\1root/g'
-    's/(DB_PASSWORD=)(.*)/\1/g'
+    "s/(DB_USERNAME=)(.*)/\1$DBUSER/g"
+    "s/(DB_PASSWORD=)(.*)/\1$BDPASSWORD/g"
```

Comment:
DB, not BD

Reply:
done! language derived typo DataBase vd. Base de Dados 😅